vms/proposervm/batched_vm_test.go

```diff
@@ -583,6 +588,112 @@ func TestBatchedParseBlockPreForkOnly(t *testing.T) {
 	require.Equal(builtBlk3.ID(), res[2].ID())
 }
 
+func TestBatchedParseBlockParallel(t *testing.T) {
+	require := require.New(t)
+
+	parentID := ids.ID{1}
```

Comment:
(fyi; no action required) `ids.GenerateTestID()` was introduced recently so you may not see it in other tests, but just letting you know that it's there.

Reply:
Used as suggested, thanks